src/Pos/Explorer/Txp/Local.hs

```diff
@@ -147,22 +153,19 @@ eTxProcessTransaction itw@(txId, TxAux {taTx = UnsafeTx {..}}) = do
 --   2. Remove invalid transactions from MemPool
 --   3. Set new tip to txp local data
 eTxNormalize ::
-       ( MonadIO m
-       , MonadBaseControl IO m
-       , MonadDBRead m
-       , MonadGState m
-       , MonadTxpMem ExplorerExtra ctx m
-       , WithLogger m
+       ( ETxpLocalWorkMode ctx m
+       , MonadSlots m
        )
     => m ()
 eTxNormalize = do
     utxoTip <- GS.getTip
+    epoch <- maybe (throwM ToilSlotUnknown) (pure . siEpoch) =<< getCurrentSlot
```

Comment:
Wait, what is it? Why is it different from `txNormalize`? It should be the same.